### PR TITLE
HDFS data migration doc

### PR DESCRIPTION
**Pull request by mzmssg**

### CODE REVIEW

File: docs/hadoop/hdfs_service.md [hunk 1]
```diff
@@ -67,10 +68,32 @@ which supports a comma-delimited list of directories to configure multiple disks
...
+    python paictl.py service stop -n hadoop-data-node
+    ```
+3. On every data node, manually copy data from *path1* to *path2*.
+4. Edit cluster-configuration *${hadoop-data-node.storage_path}* to *path2*, 
+    and [push to configmap](../pai-management/doc/push-cfg-and-set-id.md).
+5. Start hadoop data node.
+    ```bash
+    python paictl.py service start -n hadoop-data-node
+    ```
+6. Access HDFS web to confirm block status(by default *master_ip:5070*).
```
Comment (YanjieGao) [inline docs/hadoop/hdfs_service.md]: port is 50070?

File: docs/hadoop/hdfs_service.md [hunk 2]
```diff
@@ -67,10 +68,32 @@ which supports a comma-delimited list of directories to configure multiple disks
...
+If you want to reconfigure HDFS data storage path, please follow these steps to avoid data loss:
+1. Confirm HDFS path, it will be *${hadoop-data-node.storage_path}* if specified, 
+    or *${cluster.common.data-path}/hdfs/data*. Assume current cluster HDFS path is *path1*, 
+    new path is *path2*.
+2. Stop hadoop data node.
+    ```bash
+    python paictl.py service stop -n hadoop-data-node
+    ```
+3. On every data node, manually copy data from *path1* to *path2*.
+4. Edit cluster-configuration *${hadoop-data-node.storage_path}* to *path2*,
```
Comment (YanjieGao) [inline docs/hadoop/hdfs_service.md]: pls tell user in which config file and link direct to that file.  https://github.com/Microsoft/pai/blob/6e6d41da7a52fdcf5666d559281441db78c6472c/deployment/quick-start/services-configuration.yaml.template#L59

File: docs/hadoop/hdfs_service.md [hunk 3]
```diff
@@ -67,10 +68,32 @@ which supports a comma-delimited list of directories to configure multiple disks
...
-* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, otherwise in the *hdfs/data* directory under the storage path.
+* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, 
+                    otherwise in the *hdfs/data* directory under common storage path.
 * Host Configuration: Its path is defined by *host-configuration* configuration map.
 * Temp Data: It is in the *hadooptmp/datanode* directory under the storage path.
 
+## Data Migration <a name="Data_Migration"></a>
+
+If you want to reconfigure HDFS data storage path, please follow these steps to avoid data loss:
+1. Confirm HDFS path, it will be *${hadoop-data-node.storage_path}* if specified,
```
Comment (YanjieGao) [inline docs/hadoop/hdfs_service.md]: pls tell user this config on which file

File: docs/hadoop/hdfs_service.md [hunk 4]
```diff
@@ -67,10 +68,32 @@ which supports a comma-delimited list of directories to configure multiple disks
 ### Data Node <a name="Data_Node"></a>
 
 * Configuration Data: Its path is defined by *hadoop-data-node-configuration* configuration map.
-* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, otherwise in the *hdfs/data* directory under the storage path.
+* Data Storage: If  *${hadoop-data-node.storage_path}* specified, blocks are stored in these paths, 
+                    otherwise in the *hdfs/data* directory under common storage path.
 * Host Configuration: Its path is defined by *host-configuration* configuration map.
 * Temp Data: It is in the *hadooptmp/datanode* directory under the storage path.
 
+## Data Migration <a name="Data_Migration"></a>
```
Comment (YanjieGao) [inline docs/hadoop/hdfs_service.md]: Could this extend to also support other scenario: user hdfs is full, vm mount new disk, keep previous datapath and add new disk to existing HDFS. i remember currently support multipath config for hdfs

File: docs/hadoop/hdfs_service.md (hunk 2 above)
Comment (mzmssg) [reply docs/hadoop/hdfs_service.md]: added

File: docs/hadoop/hdfs_service.md (hunk 3 above)
Comment (mzmssg) [reply docs/hadoop/hdfs_service.md]: added

File: docs/hadoop/hdfs_service.md [hunk 5]
```diff
@@ -75,16 +75,32 @@ which supports a comma-delimited list of directories to configure multiple disks
 
 ## Data Migration <a name="Data_Migration"></a>
 
-If you want to reconfigure HDFS data storage path, please follow these steps to avoid data loss:
-1. Confirm HDFS path, it will be *${hadoop-data-node.storage_path}* if specified, 
+HDFS support configure multipath, 
+if just need to expand current storage, you could keep current path and append a new disk path as below:
```
Comment (YanjieGao) [inline docs/hadoop/hdfs_service.md]: if to If

File: docs/hadoop/hdfs_service.md (hunk 4 above)
Comment (mzmssg) [reply docs/hadoop/hdfs_service.md]: added

File: docs/hadoop/hdfs_service.md (hunk 5 above)
Comment (mzmssg) [reply docs/hadoop/hdfs_service.md]: done

**Pull request merged**